artemis/src/main/java/tech/pegasys/artemis/DepositCommand.java

```diff
@@ -81,44 +99,144 @@ public void generate(
           int validatorCount,
       @Option(
               names = {"--output-file", "-o"},
+              paramLabel = "<FILE>",
               description =
-                  "File to write validator keys to. Keys are printed to std out if not specified")
-          String outputFile) {
+                  "File to write unencrypted validator keys to. Keys are printed to std out if not specified")
+          String outputFile,
+      @Option(
+              names = {"--encrypt-keys", "-e"},
+              defaultValue = "false",
+              paramLabel = "<TRUE|FALSE>",
+              description = "Encrypt validator keys. (Default: false)",
+              arity = "1")
+          boolean encryptKeys,
+      @Option(
+              names = {"--keystore-output-dir", "-k"},
+              paramLabel = "<PATH>",
```

Comment:
How does this relate to `--output-file`?  Seems kind of confusing to have two separate options to specify where to output things.  @jakehaugen any ideas on the right CLI options for this?

Reply:
`--keystore-output-dir` is not used anymore. Instead I am now using `--output-path` to be treated either as directory or file depending upon we are using encrypted mode or unencrypted mode.